packages/astro/src/@types/astro.ts

```diff
@@ -1095,6 +1096,40 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		middleware?: boolean;
+		
+		/**
+		 * @docs
+		 * @name experimental.hybridOutput
+		 * Enable experimental support for opt-out prerendering
```

Comment:
```suggestion
		 * Enable experimental support for hybrid SSR with pre-rendering enabled by default.
```
I'm thinking something like this is a little more descriptive if people are reading to figure out what this is? Adjust as needed to be totally accurate, but I think this sentence could more explain what the option does in terms the users will think of their project.

Reply:
Agreed, this explains the feature better